Merge /careers and /team into /who-we-are (#99)
publiccodenet/publiccode.net: who-we-are/careers/index.md

---
redirect_from:
    - careers/index.md
---

# Careers

> Join our mission to enable public-purpose software and policy that is open and collaborative.

Reviewer: ```suggestion
Join us in enabling public-purpose software and policy that is open and collaborative.
```
